sourcemap-uploader/src/lib.ts

```diff
@@ -144,17 +144,16 @@ async function getAllSourceMapFiles(paths: string[]) {
         glob(
           "**/*.js?(.map)",
           { cwd: realPath, nodir: true, ignore: "**/node_modules/**/*" },
-          async (err, files) => {
-            for (const file of files) {
-              map.push({
-                path: join(realPath, file),
-                name: file,
-              });
-            }
-
-            resolve();
+        ).then((files) => {
+          for (const file of files) {
+            map.push({
+              path: join(realPath, file),
+              name: file,
+            });
           }
-        );
+
+          resolve();
+        });
```

Comment:
FYI, `glob >= 9.0.0` now uses promises instead of callbacks

Reply:
thanks lucas!